packages/it-multipart/index.js

```diff
@@ -143,6 +198,7 @@ function waitForStreamToBeConsumed (stream) {
         return next
       } catch (err) {
         pending.reject(err)
+        throw err
```

Comment:
Why reject and also throw?

Reply:
Because reject will fail `complete` promise, and throw will propage to whoever called `iterator.next()`. In other words if we just returned `{ done: true }` than consumer of yielded `part.body` will never know error has occurred, unless it's the same actor that is iterating over results as well and also iterates over all them.

This way if they are multiple actors both will observe the error.